### PR TITLE
Convert rst to mdx bert

### PR DESCRIPTION
**Pull request by LysandreJik**
Convert the BERT model file to mdx.

### CODE REVIEW

File: docs/source/model_doc/bert.mdx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+improvement) and SQuAD v2.0 Test F1 to 83.1 (5.1 point absolute improvement).*
+
+Tips:
+
+- BERT is a model with absolute position embeddings so it's usually advised to pad the inputs on the right rather than
+  the left.
+- BERT was trained with the masked language modeling (MLM) and next sentence prediction (NSP) objectives. It is
+  efficient at predicting masked tokens and at NLU in general, but is not optimal for text generation.
+
+This model was contributed by [thomwollf](https://huggingface.co/thomwolf). The original code can be found [here](https://github.com/google-research/bert).
```
Comment (julien-c) [inline docs/source/model_doc/bert.mdx:42]:
```suggestion
This model was contributed by [thomwolf](https://huggingface.co/thomwolf). The original code can be found [here](https://github.com/google-research/bert).
```

File: docs/source/model_doc/bert.mdx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+[[autodoc]] BertTokenizer
+    - build_inputs_with_special_tokens
+    - get_special_tokens_mask
+    - create_token_type_ids_from_sequences
+    - save_vocabulary
+
+## BertTokenizerFast
+
+[[autodoc]] BertTokenizerFast
+    - all
```
Comment (sgugger) [inline docs/source/model_doc/bert.mdx:60]:
```suggestion
[[autodoc]] BertTokenizerFast
```
all is the default, so no need to put anything.

File: docs/source/model_doc/bert.mdx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+## Bert specific outputs
+
+[[autodoc]] models.bert.modeling_bert.BertForPreTrainingOutput
+    - all
+
+[[autodoc]] models.bert.modeling_tf_bert.TFBertForPreTrainingOutput
+    - all
+
+[[autodoc]] models.bert.modeling_flax_bert.FlaxBertForPreTrainingOutput
+    - all
```
Comment (sgugger) [inline docs/source/model_doc/bert.mdx:71]: Same here

**Review by sgugger — APPROVED** [review]
Thanks for converting this file!

**Pull request merged**